extensions/resource-deployment/src/services/tools/azdataTool.ts

```diff
@@ -122,8 +126,8 @@ const defaultInstallationCommands = [
 	},
 	{
 		comment: localize('resourceDeployment.Azdata.InstallingAzdata', "installing azdata ..."),
-		command: `pip3 install -r https://aka.ms/azdata --quiet --user`
+		command: `pip3 install -r ${vscode.workspace.getConfiguration(DeploymentConfigurationKey)[AzdataPipInstallUriKey]} ${vscode.workspace.getConfiguration(DeploymentConfigurationKey)[azdataPipInstallArgsKey]} --quiet --user`
```

Comment:
introduce local constants for the user settings like: ${vscode.workspace.getConfiguration(DeploymentConfigurationKey)[AzdataPipInstallUriKey]}

Reply:
Actually I can't, I made more changes to make it responsive to the user setting changes without reloading ADS.